web/src/components/RichText.js

```diff
@@ -78,15 +87,16 @@ class RichText extends Component {
   };
 
   render() {
-    const { uploadFile: upload } = this.props;
+    const { id, onSync, uploadFile: upload } = this.props;
```

Comment:
Does this `onSync` event already exist in the elements that use the RTE? I don't see where it's provided to the editor.

Reply:
Yeah, it's a prop they pass into the `RichText` component. Without it, we'd never know the rich text changed. 😄